agent/common.sh

```diff
@@ -10,6 +10,8 @@ export AGENT_USE_APPLIANCE_MODEL=${AGENT_USE_APPLIANCE_MODEL:-"false"}
 export AGENT_APPLIANCE_HOTPLUG=${AGENT_APPLIANCE_HOTPLUG:-"false"}
 export AGENT_PLATFORM_TYPE=${AGENT_PLATFORM_TYPE:-"baremetal"}
 
+export AGENT_INSTALL_CONFIG_HOSTS=${AGENT_INSTALL_CONFIG_HOSTS:-"false"}
```

Comment:
A couple of comments:
* A little bit of name bikesheeding, but I'd prefer something like `AGENT_FORCE_INSTALL_CONFIG` (I don't have a better alternative!) to indicate that we'll try to generate a config into install-config.yaml (a general name may also be valid for other fields?)
* If true, shouldn't make sense only when AGENT_PLATFORM_TYPE == baremetal?

Reply:
Sure, I'm fine with changing the name although we need to still indicate HOSTS in that name since this is really about adding additional host info to install-config, we can still use install-config.yaml without this  so its not like we're using it to select install-config.yaml.
How about AGENT_BM_HOSTS_IN_INSTALL_CONFIG? That would also identify that these are baremetal hosts which, as you rightly pointed out, we'll need an additional check for.